frontend-react/src/pages/details/Details.tsx

```diff
@@ -26,7 +25,7 @@ function useQuery() {
 
 const DetailsContent = () => {
     let queryMap = useQuery();
-    let reportId = queryMap["reportId"];
+    let reportId = queryMap ? ["reportId"] : "";
```

Comment:
Wait, does this work the same way as `queryMap ? queryMap["reportId"] : "" `??

Reply:
I also believe I've used optional chaining like `queryMap?.["reportId"] : ""`. I love finding new little shortcuts like these!

Reply:
Good catch! Yeah, this seems like a mistake. 
Seems like: `const reportId = queryMap?.["reportId"] || "";`  would make more sense.

Reply:
Fixed.